### PR TITLE
Added logging functionality. 

### PR DESCRIPTION
**Pull request by pl9ed**
Logs are now kept in /logs/Docutest_log.txt

### CODE REVIEW

**Comment by pl9ed** [comment]
Code coverage on new code is lower, but most of it is for logging exceptions that we didn't catch/handle properly so if the program does crash, we have a stacktrace to debug with.

**Comment by IkenoXamos** [comment]
> Code coverage on new code is lower, but most of it is for logging exceptions that we didn't catch/handle properly so if the program does crash, we have a stacktrace to debug with.

If we need to spend our manpower on other tasks as of now, this could be acceptable. However, generally speaking this would not be a recommended solution. Ideally, we would like the application to not crash at all. If we can catch potential crash scenarios ahead of time with unit tests, that would be preferred.

File: pom.xml [hunk 1]
```diff
@@ -1,225 +1,236 @@
...
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.3.4.RELEASE</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.revature</groupId>
```
Comment (IkenoXamos) [inline pom.xml]: This entire file was converted to tab characters. I believe the style guide we are enforcing prefers spaces over tabs.

**Review by IkenoXamos — CHANGES_REQUESTED** [review]
The `pom.xml` file should use spaces instead of tabs.

File: pom.xml (hunk 1 above)
Comment (pl9ed) [reply pom.xml]: I forgot I didn't set my XML editor in eclipse to use spaces instead of tabs. Should be fixed now.

File: pom.xml [hunk 2]
```diff
@@ -48,8 +48,8 @@
             <artifactId>swagger-parser</artifactId>
             <version>1.0.51</version>
         </dependency>
-    
-       <dependency>
+
+        <dependency>
```
Comment (IkenoXamos) [inline pom.xml:52]: Random extra space seems to be added.

**Review by IkenoXamos — COMMENTED** [review]
It would be preferable to remove the extra space in the `pom.xml`, but aside from that, looks good.

File: pom.xml (hunk 2 above)
Comment (IkenoXamos) [reply pom.xml:52]: This does appear to actually be the correct alignment.

**Pull request merged**